programs/survival/lava_sponge.sc

```diff
@@ -0,0 +1,44 @@
+//just a sponge work on lava like water
+//by _GieR
+
+
+__config() -> (
+   m(
+      l('stay_loaded','true')
+   )
+);
+
+
+__on_player_places_block(player, item_tuple, hand, block)->(
+    if(hand != 'mainhand' || item_tuple:0 != 'sponge' || all(neighbours(pos(block)),_!='lava'), return());
+    set(pos(block),'wet_sponge');

```

Comment:
You can avoid the return call, and instead wrap the code in an if statement:
```
if(hand=='mainhand' && item_tuple:0=='sponge' && any(neighbours(pos(block)), _=='lava'),
    sponge
)
```
Thats just cos return, break and exit calls tend to be slower for scarpet parser, so it's best not to use them when possible

Reply:
k